packages/strapi-plugin-upload/admin/src/components/UploadList/Container.js

```diff
@@ -3,7 +3,7 @@ import ContainerFluid from '../ContainerFluid';
 
 const Container = styled(ContainerFluid)`
   margin-bottom: 4px;
-  padding-top: 15px;
+  padding-top: 20px;
```

Comment:
Same here, as the UploadList is used in the content manager modal, check if it works with this new value.